rdoc/gosu.rb

```diff
@@ -475,7 +475,7 @@ def subimage(left, top, width, height); end
     # @see #draw_as_quad
     # @see https://github.com/gosu/gosu/wiki/Basic-Concepts#drawing-with-colours Drawing with colors, explained in the Gosu Wiki
     # @see https://github.com/gosu/gosu/wiki/Basic-Concepts#z-ordering Z-ordering explained in the Gosu Wiki
-    def draw(x, y, z, scale_x=1, scale_y=1, color=0xff_ffffff, mode=:default); end
+    def draw(x, y, z?ß, scale_x=1, scale_y=1, color=0xff_ffffff, mode=:default); end
```

Comment:
`z?ß` shouldn't that be `z=0`?

Reply:
Whoops, patched the docs as the last thing before I went to bed. Thanks for catching it, chances are the CI wouldn't have! :+1: